test/Microsoft.Azure.WebJobs.Host.EndToEndTests/ApplicationInsights/ServiceBusRequestAndDependencyCollectionTests.cs

```diff
@@ -78,6 +79,33 @@ await host.GetJobHost()
             ValidateServiceBusRequest(sbTriggerRequest, _endpoint, _queueName, nameof(ServiceBusTrigger), operationId, sbOutDependency.Id);
         }
 
+        [Fact]
+        public async Task ServiceBusRequestWithoutParent()
+        {
+            using (var host = ConfigureHost())
+            {
+                var sender = new MessageSender(_connectionString, _queueName);
+                await sender.SendAsync(new Message { Body = Encoding.UTF8.GetBytes("message"), ContentType = "text/plain" });
```

Comment:
This test fails without your change? If so, looks great!

Reply:
you are right, this test did not repro the issue. I've fixed it -  I moved message sending to happen before AppInsights initializes in ConfigureHost and before it starts tracking calls and injects parent info into the message.

Reply:
Ok, great. So now we know we've got a good regression test.